### PR TITLE
Enhancements to lab docs

### PR DESCRIPTION
**Pull request by sabre1041**
Enhancements to lab docs

### CODE REVIEW

File: scenario5/README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Now that the root cause (or at least parts of it) has been identified, attempt t
 
 There is no need to start each service independently. The `atomic-openshift-node` systemd service has `openvswitch` as one of its dependencies, so when `atomic-openshift-node` is started, `openvswitch` service is started too.
 
-Next, check that the `openvswitch` service has been started as part of starting the `atomic-openshift-node` service. The output should show somathing similar to the example below:
+Next, check that the `openvswitch` service has been started as part of starting the `atomic-openshift-node` service. The output should show someone similar to the example below:
```
Comment (makentenza) [inline scenario5/README.md]: is not this 'something' instead 'someone'?

**Review by makentenza — CHANGES_REQUESTED** [review]
@sabre1041 everything looks good, I've got just a question regarding one particular change.

File: scenario5/README.md (hunk 1 above)
Comment (sabre1041) [reply scenario5/README.md]: fixed

**Comment by makentenza** [comment]
Thanks @sabre1041 LGTM

**Pull request merged**